src/core/FileUpload/FileUpload.test.tsx

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { FileUpload } from './FileUpload';
+
+it('should render in its most basic state', () => {
+  const mockContent = 'Mock drag content';
+  const mockChildren = 'Mock children to wrap';
+  const { container } = render(
+    <FileUpload content={mockContent} onFileDropped={jest.fn}>
+      {mockChildren}
+    </FileUpload>,
+  );
+
+  const component = container.querySelector('.iui-file-upload') as HTMLElement;
+  expect(component).toBeTruthy();
+
+  const content = container.querySelector('.iui-content') as HTMLElement;
+  expect(content).toBeTruthy();
+  expect(content.textContent).toEqual(mockContent);
+
+  const children = component.lastChild as HTMLElement;
+  expect(children).toBeTruthy();
+  expect(children.textContent).toEqual(mockChildren);
+});
```

Comment:
@veekeys @bentleyvk Any idea how I would actually test the drag behavior? Just mock events?

Reply:
Looks like you need to simulate events yourself as jsdom does not support them:
https://react-dnd.github.io/react-dnd/docs/testing#simulate-the-dom
![image](https://user-images.githubusercontent.com/36186912/116656131-d6448200-a994-11eb-87a1-5baefcd26d4f.png)